Day-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,81 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ButtonContainer from "./components/ButtonComponents/ButtonContainer";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+class App extends React.Component {
+  render() {
+return(
+    <div className="calculator-wrapper">
+      <CalculatorDisplay />
+      <ButtonContainer className="clear-button">
```

Comment:
The way your using ButtonContainer here is for presentation. A div with `className="clear-button"` would be better. Then keep `<ActionButton className="three-text" text="clear" />` and `<NumberButton bgColor="red-button" text="/" />` in the same place. You won't need to use props.children. To pass down classNames/styles, do what you've been doing with attributes like `text="clear"`.